### ISSUE TITLE
In the repo you are using radix-ui insted of shadcn

### ISSUE THREAD

**Issue by securi404** [piyush-eon/url-shortener#4]
Please fix the issue with package you are usinf shadcn but in package.json you have radix-ui

**Comment by piyush-eon** [comment]
So whats the issue? Please check shadcn’s docs

**Comment by shivang21007** [comment]
@sgcocac  Don't be confuse, these are Shadcn components which under the hood uses multiple ui packages .

**Comment by securi404** [comment]
> So whats the issue? Please check shadcn’s docs

**
![image](https://github.com/user-attachments/assets/8a7bb2af-d0fb-47c7-8fef-5d35368f47e8)
**

In your tutorial you are using shadcn and in repository i have checked in package.json you have radix-ui.

So, please update your documentation or should I? If you want me to create a documentation for that I am happy to provide that for this repo. And I have completed the tutorial and thanks to you I have learned some new technology.

**Comment by piyush-eon** [comment]
Bro radix ui is a part of shadcn ui. Have you not used shadcn ui before? Its not a package

**Comment by securi404** [comment]
> @sgcocac Don't be confuse, these are Shadcn components which under the hood uses multiple ui packages .

Yes, you are correct. But, ShadCN UI is a set of components built on Radix UI, and while you are using several Radix UI components (e.x., @radix-ui/react-accordion, @radix-ui/react-dialog, etc.), there is no direct indication that you're using ShadCN and I followed your tutorial it gave me errors because of unclear documentations. 

I understand you are thinking I am just wasting time here, But updating/upgrading the code might help so I raised the issue. It should have been so simple if I explained it for the first time. Hope you understand bye.
